mmtrack/datasets/dataset_wrappers.py

```diff
@@ -32,6 +33,13 @@ def __init__(self, dataset_cfgs, dataset_sampling_weights=None):
             ]
 
         datasets = [build_dataset(cfg) for cfg in dataset_cfgs]
+        self.CLASSES = datasets[0].CLASSES
+        if hasattr(datasets[0], 'flag'):
+            flags = []
+            for i in range(0, len(datasets)):
+                flags.append(datasets[i].flag)
+            self.flag = np.concatenate(flags)
```

Comment:
why add `flag` attribute?

Reply:
the `isinstance(dataset, BaseSOTDataset)` in `build_dataloader` need it.
I have tried an alternative.